test/integration/publishing_test.rb

```diff
@@ -18,11 +18,11 @@ class PublishingTest < ActiveSupport::TestCase
       # edition has been published
       public_updated_at: Time.zone.now.as_json
     )
-    request = stub_publishing_api_put_item(@presenter.base_path, expected_attributes)
+    requests = stub_publishing_api_put_content_links_and_publish(expected_attributes)
 
     perform_force_publishing_for(@draft_edition)
 
-    assert_requested request
+    requests.each { |request| assert_requested request }
```

Comment:
If the method lives within `whitehall` or `gds-api-adapters`, I'd suggest `assert_all_requested(array)`. If it were contributed back to `webmock`, I'd just allow passing an array into `assert_requested` itself.